packages/amplify-nodejs-function-runtime-provider/src/__tests__/utils/__snapshots__/execute.test.ts.snap

```diff
@@ -565,4 +565,8 @@ exports[`execute callback handler should fail when returning an error 1`] = `[Er
 
 exports[`execute callback handler should succeed with a returned value 1`] = `"foo"`;
 
+exports[`execute failed execution should return the proper error 1`] = `[ReferenceError: undedvar is not defined]`;
```

Comment:
shouldn't this error message also include the stacktrace?

Reply:
Updated stacktrace is included added a different test suite for this